akvo/rest/views/indicator.py

```diff
@@ -12,7 +12,10 @@
 class IndicatorViewSet(PublicProjectViewSet):
     """
     """
-    queryset = Indicator.objects.all()
+    queryset = Indicator.objects.all().select_related(
```

Comment:
Do the same for the IndicatorFrameworkViewSet maybe? I don't know if it's used anywhere, but it should suffer from the same problem.